README.md

```diff
@@ -66,21 +64,21 @@ The `Runner` constructor returns a new instance of `Runner`.
 ```js
 import { Runner } from 'gallinago';
 
-const runner = new Runner();  // pass true to the constructor to enable stdout
+const runner = new Runner();  // pass true to the constructor to print stdout and stderr
 ```
 
 #### Options
 
 `Runner` takes two boolean flags (`true`|`false`)
-- Standard Out - pass `true` to have the Runner log to `stdout`
+- Enable Debug - pass `true` to have the Runner log `stdout` and `stderr`
```

Comment:
I've renamed this for clarity. Not a breaking change because it's just using positional params